src/portal/lib/src/vulnerability-scanning/scanning.scss

```diff
@@ -144,4 +135,40 @@ hr{
 
 .help-icon {
     margin-left: 3px;
-}
\ No newline at end of file
+}
+
+.mt-3px {
+    margin-top: 5px;
+}
+
+.label-critical {
+    background:red;
+    color:#621501;
+    border:1px solid #f8b5b4
+}
+
+.label-danger {
+    background:#e64524!important;
```

Comment:
Why all need !important? It need very careful to use !important.

Reply:
label-danger is a CLR  class name, I have to add !important to rewrite it